modeldict/base.py

```diff
@@ -219,3 +231,8 @@ def _cleanup(self, *args, **kwargs):
         # We set _last_updated to a false value to ensure we hit the
         # last_updated cache on the next request
         self._last_checked_for_remote_changes = 0.0
+
+    def _cache_set_kwargs(self):
```

Comment:
I think with would be a good place to use `@property`

Reply:
Fixed